pkg/hub/localizer/localizer_test.go

```diff
@@ -0,0 +1,412 @@
+package localizer
+
+import (
+	"github.com/ghodss/yaml"
+	"reflect"
```

Comment:
Please format the imports. Still outstanding.
Please don't make this resolved unless it is addressed.

Reply:
Again.
Please add license here.